db/migrate/20170217194851_create_firmwares.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFirmwares < ActiveRecord::Migration[5.0]
+  def change
+    create_table :firmwares do |t|
+      t.string   :name
+      t.string   :build
+      t.string   :version
+      t.string   :ph_server_uuid, :index => true
```

Comment:
- What is this?  Is this another object in our database, and if so, should it just be a foreign key?  Is this a unique manager reference, and if so, should this be an ems_ref or maanger_ref column?
- What object is this attached to?  I would always expect an ems_id (or manager_id) on every new object.  Once you have that, I would expect that that ems_id, ems_ref would be a joint index.